pyro/generic/generic.py

```diff
@@ -27,6 +27,12 @@ def __getattribute__(self, name):
         return getattr(module, name)
 
 
+@contextmanager
+def model(rng):
```

Comment:
1. nit: the arg is a seed, not an rng, right? rng usually means an object with full rng state.
2. is it possible to define this in a way where a seed need not be specified by default, so as to exactly reproduce current vanilla pyro behavior?

Reply:
1. That's bad naming, it should just be named `seed` or `rng_seed`. Will change. 
2. Sure. I think we can even remove this contextmanager now, since we can just use `handlers.seed(...)` instead, which will not have an effect for Pyro.